inspirehep/modules/theme/bundles.py

```diff
@@ -71,12 +71,6 @@
     }
 )
 
-detailedjs = NpmBundle(
-    "js/detailed_record_init.js",
-    filters="requirejs",
-    output="gen/detailed_record.%(version)s.js",
-)
```

Comment:
Why is this removed? seems to be still in `setup.py` https://github.com/inspirehep/inspire-next/pull/1699/files#diff-2eeaed663bd0d25b7e608891384b7298R239